web/ui/react-app/src/pages/graph/MetricsExplorer.tsx

```diff
@@ -1,33 +1,57 @@
-import React, { Component } from 'react';
-import { Modal, ModalBody, ModalHeader } from 'reactstrap';
+import React, { Component, ChangeEvent } from 'react';
+import { Modal, ModalBody, ModalHeader, Input } from 'reactstrap';
+import { Fuzzy, FuzzyResult } from '@nexucis/fuzzy';
+const fuz = new Fuzzy({ pre: '<strong>', post: '</strong>', shouldSort: true });
 
 interface Props {
   show: boolean;
   updateShow(show: boolean): void;
   metrics: string[];
   insertAtCursor(value: string): void;
 }
-
-class MetricsExplorer extends Component<Props> {
+type MetricsExplorerState = {
```

Comment:
Style nit: add empty lines before and after such a top-level definition (so one between lines 11+12 and one between lines 14+15).

Reply:
should be fixed in last commit